resources/views/components/layouts/head/head.blade.php

```diff
@@ -6,12 +6,14 @@
 <meta name="csrf-token" content="{{ csrf_token() }}">
 <meta name="pusher-key" content="{{ config('broadcasting.connections.pusher.key') }}">
 <meta name="pusher-cluster" content="{{ config('broadcasting.connections.pusher.options.cluster', 'mt1') }}">
+<meta name="currency-code" content="{{ \FluxErp\Models\Currency::default()?->iso }}">
```

Comment:
Is this ever used anywhere? I don't see it, but I may also be blind.

Reply:
yes, if no currency is available for a formatter this tag is used as currency.